doc/reference/pipeline_spec.md

```diff
@@ -575,6 +597,17 @@ https://github.com/<your_org>/<your_repo>/settings/hooks/new
 ```
 Or navigate to webhooks under settings. Then you'll want to copy the `Githook URL` into the 'Payload URL' field.
 
+### Spout
```

Comment:
Similarly, I would move this description to immediately after the description of services (ie. after the paragraphed headed `Service (alpha feature, optional)`) 

Also be sure to mention that it is optional, and that either `Spout` OR `Service` can be set, but NOT both.